backend/src/main/java/com/github/EazyErik/Repository/DrinksRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.github.EazyErik.Repository;
+
+
+import lombok.Data;
+import org.springframework.stereotype.Repository;
+
+@Repository
+
+public interface DrinksRepository{
```

Comment:
The repository should extend an interface in the spring frameword, for example `MongoRepository<Drink, String>`.